templates/_init/flake.nix

```diff
@@ -1,19 +1,12 @@
 {
-  
+  description = "Floxpkgs/Project Template";
+  nixConfig.bash-prompt = "[flox] \\[\\033[38;5;172m\\]λ \\[\\033[0m\\]";
+  inputs.floxpkgs.url = "github:flox/floxpkgs";
 
   # Declaration of external resources
   # =================================
 
   # =================================
 
-
-  description = "Floxpkgs/Project Template";
-  nixConfig.bash-prompt = "[flox] \\[\\033[38;5;172m\\]λ \\[\\033[0m\\]";
-
-  # Template DO NOT EDIT
-  # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-  # TODO: injected by the cli, or used via registry?
-  inputs.floxpkgs.url = "github:flox/floxpkgs";
-  outputs = args @ {floxpkgs, ...}: floxpkgs.capacitor args (import ./flox.nix);
-  # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+  outputs = args @ {floxpkgs, ...}: floxpkgs.project args (_: {});
```

Comment:
Perhaps place some comments between the “{}”. Eventually we’d provide a reference to the capacitor API for people to select their own plugins or do advanced overloading or otherwise access the passthru escape hatch.

Reply:
Yes, I left it empty for now since we don't have documentation on the contents of that.
Pointing people (even nixers) into a void didn't seem to be a good idea to me